### ISSUE TITLE
dataset problem

### ISSUE THREAD

**Issue by suojifeng** [hanzhu97702/ISPRS_STMA#1]
The dataset given needs to be processed by myself, right? thank you

**Comment by hanzhu97702** [comment]
This public dataset has been pre-processed. Since it is a complete remote sensing scene, you need to cut and split small patches according to your requirement.

**Comment by suojifeng** [comment]
> This public dataset has been pre-processed. Since it is a complete remote sensing scene, you need to cut and split small patches according to your requirement.

yes，I asked because I don't have this file. Sorry data_file = './S1_2017_2018/train_list.npz'

**Comment by XuanfengLiuNo1** [comment]
怎样通过 label.mat VH.mat VV.mat 获得train_list.npz？train_list.npz的形状应该是什么？谢谢

**Comment by hanzhu97702** [comment]
The file of 'dataset_prepare.py' has been uploaded to generate train_list.npz.

**Comment by 11144477** [comment]
Hello, I saw dataset_prepare when setting up the dataset, the training set and the test set are distinguished by different coverages,? So are the training set and the test set independent of each other???thank you

**Comment by 11144477** [comment]
Hello, I saw dataset_prepare when setting up the dataset, the training set and the test set are distinguished by different overlays. So are the training set and the test set independent of each other???
